### DISCUSSION TITLE
Web Test Runner: Preventing loading of script tags

### DISCUSSION THREAD

**Discussion by chrischrischris** [modernweb-dev/web#2399] [Q&A]
Hi All - First off thanks for making/maintaining a great test framework!

I can't figure out how to stop <script> tags from loading that are injected onto the page.  Using middleware and/or import maps I can change the files imported by modules but nothing seems to stop script tags from loading.

We have a lot of external dependencies and so what's happening is on every test they are loaded.  I'm trying to get our tests to not load anything other than from localhost.

Thanks!

**Comment by thepassle** [comment]
You could transform the index.html in a middleware and take the script out there

**Reply by chrischrischris** [reply]
The script tags are inserted dynamically via js, so the script tags are not available to remove.
